src/MongoDB.Bson/Properties/AssemblyInfo.cs

```diff
@@ -14,21 +14,10 @@
 */
 
 using System;
-using System.Reflection;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
```

Comment:
The AssemblyInfo.cs files only have a few things left in them. Only those that aren't controlled by MSBuild from properties in the .csproj files.